inc/ticket.class.php

```diff
@@ -1349,9 +1349,8 @@ function post_updateItem($history=1) {
          }
       }
 
-      $this->updates[] = "actiontime";
-
       if (count($this->updates)) {
+         $this->updates[] = "actiontime";
```

Comment:
the next part seems to be also strange as it test the presence actiontime in updates